File: src/main/java/com/example/demo/models/Note.java
package com.example.demo.models;

public class Note {
    private String text;

    public Note() {}

    public Note (String newText) {
        this.text = newText;
    }

    public String getText() {
        return text;
    }

    public void setText(String text) {
        this.text = text;
    }
}

File: src/main/java/com/example/demo/models/Note.kt
package com.example.demo.models
import jakarta.persistence.*
import java.time.LocalDateTime
@Entity
class Note {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Long? = null
    var text: String? = null

    constructor(id: Long?, text: String?) {
        this.id = id
        this.text = text
    }
}